frontend/src/pages/projects/screens/detail/notebooks/NotebookList.tsx

```diff
@@ -27,7 +27,7 @@ const NotebookList: React.FC = () => {
   const projectName = currentProject.metadata.name;
   const isNotebooksEmpty = notebooks.length === 0;
 
-  useRefreshInterval(FAST_POLL_INTERVAL, () =>
+  useRefreshInterval(FAST_NOTEBOOK_START_STOP_POLL_INTERVAL, () =>
```

Comment:
dont create a new poll interval, just reuse FAST_POLL_INTERVAL

Reply:
fast is the fasted we want to _poll_, if we need something faster we should instead look to websockets. In this case we should just stick to the fast poll interval